app/src/main/res/values/strings.xml

```diff
@@ -458,6 +458,10 @@
     <!-- Wallpapers -->
     <!-- Content description for various wallpapers. The first parameter is the name of the wallpaper -->
     <string name="wallpapers_item_name_content_description">Wallpaper Item: %1$s</string>
+    <!-- Snackbar message for when wallpaper is selected -->
+    <string name="wallpaper_updated_snackbar_message">Wallpaper updated!</string>
+    <!-- Snackbar label for action to view selected wallpaper -->
+    <string name="wallpaper_updated_snackbar_action">VIEW</string>
```

Comment:
I am wondering if we already have an existing `VIEW` string for the snackbar that can be used here?

Reply:
We have this one which I could make uppercase at runtime:
```
<!-- Collections -->
...
<!-- Snackbar action to view the collection the user just created or updated -->
    <string name="create_collection_view">View</string>
```

I could move it under `<!-- Snackbar -->` and make the comment more general, but I worry that context could be lost.

Reply:
Ok, let's not do that and just keep it as-is. However, now I am think the string should be changed to "View" and we add  should add the equivalent of `android:textAllCaps="true"` to the Compose Text().